autogpt_platform/backend/backend/blocks/github/issues.py

```diff
@@ -13,6 +13,7 @@
 )
 
 
+# --8<-- [start:GithubCommentBlock]
```

Comment:
This allows to pull code fragments into docs? Nice, what allows that?

Reply:
https://facelessuser.github.io/pymdown-extensions/extensions/snippets/

Reply:
See also https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/